app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,80 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.UserNotFoundException;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import com.codesoom.assignment.dto.UserData;
+import com.github.dozermapper.core.Mapper;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+
+/**
+ * 사용자 도메인의 서비스 레이어.
```

Comment:
설명이 추상적인 것 같아요. 추상적인 것 자체는 괜찮은데 그래서 뭐하는 클래스냐 하는 질문에 대한 답으로 적절한 것 같지는 않아요.

구체적으로는 다음과 같이 작성할 수 있겠죠. 이 클래스의 역할을 3초만에 이해할 수 있습니다.
```suggestion
 * 사용자 정보의 조회, 저장 등을 담당합니다.
```

이보다 더 잘 작성하실 수도 있을 거에요.